.ci/StarterRuleSet-AllRulesByCategory.groovy.txt

```diff
@@ -134,7 +134,8 @@ ruleset {
     // rulesets/dry.xml
     DuplicateListLiteral
     DuplicateMapLiteral
-    DuplicateNumberLiteral
+    //DuplicateNumberLiteral  // it is ok to use duplication, every number
```

Comment:
I do not know why we allowed trailing comments previously, please place explanation of rule suppression above rule name. Can we fix others in a separate commit?

Reply:
Done.